client/verta/tests/registry/model_version/test_artifacts.py

```diff
@@ -57,41 +57,6 @@ def test_add_artifact_file(self, model_version, in_tempdir, random_data):
         retrieved_file = model_version.get_artifact("file")
         assert retrieved_file.getvalue() == FILE_CONTENTS
 
-    def test_download(self, model_version, strs, in_tempdir, random_data):
-        key = strs[0]
-        filename = strs[1]
-        new_filename = strs[2]
-        FILE_CONTENTS = random_data
-
-        # create file and upload as artifact
-        with open(filename, "wb") as f:
-            f.write(FILE_CONTENTS)
-        model_version.log_artifact(key, filename)
-        os.remove(filename)
-
-        # download artifact and verify contents
-        new_filepath = model_version.download_artifact(key, new_filename)
-        assert new_filepath == os.path.abspath(new_filename)
-        with open(new_filepath, "rb") as f:
-            assert f.read() == FILE_CONTENTS
-
-        # object as well
-        obj = {"some": ["arbitrary", "object"]}
-        model_version.log_artifact(key, obj, overwrite=True)
-        new_filepath = model_version.download_artifact(key, new_filename)
-        with open(new_filepath, "rb") as f:
-            assert pickle.load(f) == obj
```

Comment:
Identical to https://github.com/VertaAI/modeldb/pull/2603/commits/a1810f318bfa3bcd579656ba46f9e110d6009280#diff-1a5838d629f2008e901066b48a5126a4c23f67ddf53f5dea2c69ab6433475ce8R202-R225